streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamKStreamOuterJoinTest.java

```diff
@@ -438,13 +438,13 @@ public void testOrdering() {
             inputTopic1.pipeInput(1, "A1", 100L);
             processor.checkAndClearProcessResult();
 
-            // push one item to the other window that has a join; this should produce non-joined records with a closed window first, then
-            // the joined records
-            // by the time they were produced before
+            // push one item to the other window that has a join;
+            // this should produce the joined record first;
+            // then the not-joined record
```

Comment:
This change to the comment needs to be rolled back, right? We indeed produce the left-null join result first.

Reply:
Modified the comments according to the results produced.